coordinator/src/routes/index.ts

```diff
@@ -2,7 +2,7 @@ import convertPrescriptionRoutes from "./debug/convert"
 import validatorRoutes from "./debug/validate"
 import doseToTextRoutes from "./debug/dose-to-text"
 import preparePrescriptionRoutes from "./prescribe/prepare"
-import processPrescriptionRoutes from "./process"
+import processPrescriptionRoutes from "./prescribe/process"
```

Comment:
think this should stay at top level since `process-message` serves all of: prescription-order, prescription-order-update and dispense-notification